packages/data-stores/src/starter-designs-queries/constants.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Standard themes that are verticalizable
+ */
+export const STANDARD_THEMES_V13N_WHITELIST = [
```

Comment:
Nit: `VERTICALIZABLE_STANDARD_THEMES`? (to make it consistent with the `verticalizable` field name)